### PR TITLE
Add some documentation for 1.4 release

### PR DESCRIPTION
**Pull request by jeffschriber**
## Description
Updates the docs with any undocumented new features in 1.4

## Todos

- [x] #1491 (ATM)
- [x] #842 (UHF Hess)
- [x] #1803 (E(30)ind bug fix)
- [x] #2056 (MBIS volumes)
- [x] #2081 (SAPT-D)
- [x] #2127 (MBIS volume ratios)

- [x] #1934 (cct3 plugin)
- [x] #1661 (mp2-d gradients)

I still have #1721 (Libint2 and shell screening), #1723 (QCSchema Interface), and #1728 (QCSchema Wavefunction Quantities) as being undocumented or incompletely documented. Let me know if I'm wrong here, and any suggestions (or volunteers) on how to complete them are welcome.

## Status
- [x] Ready for review
- [ ] Ready for merge

### CODE REVIEW

File: doc/sphinxman/source/oeprop.rst [hunk 1]
```diff
@@ -180,8 +180,15 @@ Minimal Basis Iterative Stockholder
 
 The Minimal Basis Iterative Stockholder (MBIS) method is one of many procedures
 that partitions a molecular one-particle density matrix into atomic electron densities.
-Running MBIS in |PSIfour| will calculate atomic charges, as well as dipoles, quadrupoles, and
-octupoles. The allowed number of iterations and convergence criteria for the stockholder 
+Running MBIS in |PSIfour| will calculate atomic valence charge widths, volume ratios,
+atomic charges, as well as dipoles, quadrupoles, and octupoles. 
+Additionally, all expectation values of radial moments of n-th order (:math:`<r^n>`) 
+are computed up to fourth order. Higher moments can be computed by specifying |globals__MAX_RADIAL_MOMENT|.
```
Comment (loriab) [inline doc/sphinxman/source/oeprop.rst:186]:
```suggestion
are computed up to fourth order. Higher moments can be computed by specifying |globals__max_radial_moment|.
```
pretty sure those substitutions haven't been trained to be caps tolerant

File: doc/sphinxman/source/sapt.rst [hunk 2]
```diff
@@ -59,6 +59,10 @@ SAPT: Symmetry-Adapted Perturbation Theory
    wherever possible, according to literature recommendations.
    In early July 2016, some total SAPT energy psivars were renamed.
 
+.. caution:: February 7, 2020, a missing term in :math:`E^{(30)}_{ind}` was added, causing
+   possible discrepancies with prior versions of the code on the order of
+   0.01 kcal/mol.
```
Comment (loriab) [inline doc/sphinxman/source/sapt.rst:64]:
```suggestion
   0.01 kcal/mol. See https://github.com/psi4/psi4/issues/1677
```

**Review by loriab — APPROVED** [review]
lgtm! Thanks so much for working on docs.

**Comment by JonathonMisiewicz** [comment]
Is there a reason this PR is marked not ready for review?

**Comment by jeffschriber** [comment]
No, Its ready for review. I'm not entirely sure how to handle docs (if they are even needed, they are more or less under the hood) for those last three outstanding PRs from the google doc that I reference above. I'm happy to add more to this PR if anything is needed

**Comment by JonathonMisiewicz** [comment]
`cct3` and `mp2-d` don't seem to be covered in your commits, despite being marked as "newly covered" in your topic post.

**Comment by jeffschriber** [comment]
There are no new docs for those, but I had them as open items when I opened the PR. I spoke with Lori about them since, and we agreed that listing them in "What are the add-on capabilities for Psi4 and what are their dependencies?" and including links to repos was sufficient. I checked them to indicate their documentation status was resolved (though I agree the check alone indicates some new doc was added)

**Review by JonathonMisiewicz — APPROVED** [review]

**Review by jturney — APPROVED** [review]

**Review by hokru — APPROVED** [review]

**Pull request merged**